src/features/instance/applications/new/ImportProjectForm.tsx

```diff
@@ -53,23 +51,23 @@ export function ImportProjectForm({
 	const submitForm = async (formData: DeployComponentFormData) => {
 		deployNewApplication({ ...formData, ...instanceParams }, {
 			onSuccess: () => {
-				toast.success(`Application ${formData.newApplicationName} created successfully`);
-				triggerRestart();
+				toast.success(`Application ${formData.applicationName} imported successfully`);
+				onRestartedSuccessfully();
```

Comment:
Deploying the new app does a rolling restart already, so all we need to do is trigger the cache invalidation and toast!